internal/tiltfile/tiltfile_test.go

```diff
@@ -56,12 +56,12 @@ func TestGetManifestConfig(t *testing.T) {
 	dockerfile := tempFile("docker text")
 	file := tempFile(
 		fmt.Sprintf(`def blorgly():
-  image = build_docker_image("%v", "docker-tag", "the entrypoint")
-  image.add(local_git_repo('.'), '/mount_points/1')
-  print(image.file_name)
-  image.run("go install github.com/windmilleng/blorgly-frontend/server/...")
-  image.run("echo hi")
-  return k8s_service("yaaaaaaaaml", image)
+		image = build_docker_image("%v", "docker-tag", "the entrypoint")
```

Comment:
My editor was being really weird with these indents

Reply:
i'm surprised skylark even accepts this as valid code bc it has tabs